ControlMeasurements/Views/Measure/Index.cshtml

```diff
@@ -0,0 +1,52 @@
+﻿@model IEnumerable<ControlMeasurements.Models.MeasurementCategory>
+
+@{
+    ViewData["Title"] = "Index";
+}
+
+
+<h2>Index</h2>
+
+<p>
+    <a asp-action="Create">Create New</a>
+</p>
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.PlaceType)
+                @Html.DisplayNameFor(model=>model.MeasurementType)
+                @Html.DisplayNameFor(model=>model.Value)
+                @*<select class="form-control" id="MeasurementType" name="MeasurementType">
```

Comment:
Jeśli masz tu kod, którego nie używasz, to lepiej go usunąć niż wykomentować :)

Reply:
To rowniez z poprzedniego, wiec juz jest usuniete :)